### DISCUSSION TITLE
Transfer Call Hangup

### DISCUSSION THREAD

**Discussion by AravindBollabathini** [versatica/JsSIP#863] [Q&A]
Iam Already built the web dialer project, Included the transfer call functionality in my code using JSSIP refer method.
I have a doubt that when we make a transfer call using refer but if that use didn't answer call how to hangup that call using response that we get from transfer call request.

**Comment by djon198360** [comment]
Тоже бы хотелось узнать ответ , как понять что вызов принят .
Событие requestSucceeded возникает когда абонент может принять звонок , а не когда он его принял .
Хотелось бы так :
Получил звонок от абонента А, с помощью метода refer перевел его на абонента Б , и после того как абонент Б поднял трубку , завершил сессию оставив абонентов А и Б .
